src/main.rs

```diff
@@ -32,12 +37,17 @@ struct Args {
 
 fn main() {
     let args = Args::parse();
-    let bytecode = args.bytecode;
 
-    let bytecode = if let Some(stripped) = bytecode.strip_prefix("0x") {
```

Comment:
Could you please leave this `strip_prefix()` after that the bytecode has been taken either from the command line, or from the file please ? This allows us to leave it as it may feel more natural !

Reply:
Should we allow users to give in both kind of inputs for a slightly improved dx? I can add that in.